Client/src/Pages/Incidents/index.css

```diff
@@ -0,0 +1,33 @@
+.maintenance-checklist-main {
+  max-width: 350px;
+  min-height: 390px;
+  margin: auto;
+  padding: 5px;
+}
+
+.maintenance-image {
+  width: 280px;
+  height: 150px;
+  background-color: whitesmoke;
+  border-radius: 10px;
+}
+
+.maintenance-title {
+  font-size: 20px;
+  font-weight: bold;
+  margin-top: 40px;
+  margin-bottom: 30px;
+}
+
+.checklist-item {
+  display: flex;
+  margin-bottom: 10px;
```

Comment:
The font color of the text should be `#475467` and a size of `13px`. Please see the Figma design.

Reply:
Thank you.